### PR TITLE
Redesiged layout.

### PR DESCRIPTION
**Pull request by yuinchien**

### CODE REVIEW

File: css/theme.css [hunk 1]
```diff
@@ -30,3 +31,61 @@ a {
...
+  z-index: -1;
+  position: absolute;
+}
+
+#section-features svg {
+  fill: white;
+  width: 40px;
+  height: 40px;
+  margin: 10px;
+}
```
Comment (frankiefu) [inline css/theme.css]: Do you think we need the same `#section-features` styles for the other sites (lit-html and LitElement)?

File: css/theme.css [hunk 2]
```diff
@@ -30,3 +31,61 @@ a {
...
+
+#browser-thumbnails img {
+  margin: 0 16px 0 0;
+}
```
[the remark this answers is not in the document]
Comment (yuinchien) [reply css/theme.css]: Good call!

File: css/theme.css (hunk 1 above)
Comment (yuinchien) [reply css/theme.css]: Possibly. Let's move it to main.css

**Comment by yuinchien** [comment]
@frankiefu changed pushed. Please take a look.

File: css/theme.css [hunk 3]
```diff
@@ -30,3 +31,24 @@ a {
...
+  }
+  #section-features .responsive-row > div:before {
+    width: 52px;
+    height: 52px;
+  }
+  #section-features svg {
+    width: 32px;
+    height: 32px;
+  }
+}
```
Comment (frankiefu) [inline css/theme.css]: Do you need to move those styles but for smaller screen to `main.css` too?

Comment (yuinchien) [reply css/theme.css]: Yes. I forgot... Just updated css and pushed.

**Pull request merged**